src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && <p className="Person__age">I am {age}</p>}
+      {!isMarried && <p className="Person__partner">I am not married</p>}
+      {isMarried && sex === 'm' && (
+        <p className="Person__partner">{partnerName} is my wife</p>
+      )}
+      {isMarried && sex === 'f' && (
```

Comment:
Let's simplify this one even more:
create a constant for a partnerGender -   `const partnerGender = sex === 'm' ? 'wife' : 'husband'`

Modify jsx:
```
      <p className="Person__partner">
        {isMarried
          ? `${partnerName} is my ${partnerGender}`
          : 'I am not married'}
      </p>
```